### ISSUE TITLE
CLI - Not including the schema in -d parameter results in critical error

### ISSUE THREAD

**Issue by CirKu17** [spider-rs/spider#150]
I installed the CLI via cargo in Ubuntu 22.04. 
If I run `spider -v --domain example.com crawl` I get

```
thread 'main' panicked at /home/inelemento/.cargo/registry/src/index.crates.io-6f17d22bba15001f/spider_cli-1.48.3/src/main.rs:57:10:
called `Result::unwrap()` on an `Err` value: Kind(NotFound)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
the crawler launches if I use http://example.com as argument.

Since the parameter is called domain and not URL this can be misleading. 
Also I would expect the error to be reported with an user-friendly error message.

Cheers and keep up the good work.

**Comment by j-mendez** [comment]
Thank you for the issue @CirKu17. We could probably auto append https:// if the domain is set without to prevent the error on the unwrap, along with displaying a better error message. I agree on changing the arg from domain to url 👍.

**Comment by j-mendez** [comment]
The main issue is fixed in `v1.48.4`, you can enable log output with `RUST_LOG=info`. If http is omitted it is not added to the url. Going to keep the issue open until the flag rename.

**Comment by j-mendez** [comment]
fixed https://github.com/spider-rs/spider/releases/tag/v1.49.0
